README.md

```diff
@@ -229,13 +229,13 @@ with the help of Trixi.jl, please cite the following articles:
 
 In addition, you can also refer to Trixi.jl directly as
 ```bibtex
-@misc{schlottkelakemper2020trixi,
+@misc{schlottkelakemper2025trixi,
   title={{T}rixi.jl: {A}daptive high-order numerical simulations
          of hyperbolic {PDE}s in {J}ulia},
   author={Schlottke-Lakemper, Michael and Gassner, Gregor J and
-          Ranocha, Hendrik and Winters, Andrew R and Chan, Jesse},
-  year={2021},
-  month={09},
+          Ranocha, Hendrik and Winters, Andrew R and Chan, Jesse
+          and Rueda-Ramírez, Andrés},
```

Comment:
For uniformity, perhaps this should also be in alphabetic order to match the `CITATION.bib` text

Reply:
As noted above, historically this was in order of joining the group of Trixi.jl PIs. For this PR, I'd keep it this way.